dev/com.ibm.ws.cdi.1.2.weld/src/com/ibm/ws/cdi/impl/weld/package-info.java

```diff
@@ -0,0 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+@org.osgi.annotation.versioning.Version("1.0")
+package com.ibm.ws.cdi.impl.weld;
```

Comment:
don't use this same package name, use a new one

Reply:
This is adding a missing package info file for the weld package.